### ISSUE TITLE
"Submit for Review" and "Return to Author" buttons not working when dataset has more than 10 files

### ISSUE THREAD

**Issue by scolapasta** [IQSS/dataverse#5990]
In a dataset with more than 10 files (i.e. a file table with pagination) the new "Submit for Review" and "Return to Author" break the "Files per Page" dropdown (the value 10 stays in the dropdown, while 25 and 50 pop out). This in turn breaks the "Submit for Review" and "Return to Author"; when you press them, you get the popup, then the spinner, then nothing happens.

A workaround until this is fixed is to click on the paginator; doing that fixes the dropdown, and then the buttons work again.

**Comment by mheppler** [comment]
Assuming this was seen in production (v. 4.15). Replicated this locally on an old 4.15 branch. Confirmed it is fixed in `develop` branch, after #5958 was merged.

Also, this same issue was reported in #4838, which was also a symptom of the same JSF rendering bug.

@scolapasta please confirm and close, or provide more detail if the problem persists.

**Comment by djbrooke** [comment]
Closing as a duplicate of #5958
